client/landing/subscriptions/components/add-sites-form/add-sites-form.tsx

```diff
@@ -29,12 +31,14 @@ type SubscriptionError = {
 const AddSitesForm = ( {
 	placeholder,
 	buttonText,
+	pathname,
 	source,
 	onChangeFeedPreview,
 	onChangeSubscribe,
 }: AddSitesFormProps ) => {
 	const translate = useTranslate();
 	const [ inputValue, setInputValue ] = useState( '' );
+	const [ searchTerm, setSearchTerm ] = useState( getUrlQuerySearchTerm( pathname ) );
```

Comment:
So these ever need to be different values? It seems below in `onTextFieldChange` we set them both to the same value. Could we consolidate these 2 state variables?

Reply:
Removed the state in [7b566f0](https://github.com/Automattic/wp-calypso/pull/101625/commits/7b566f0ed7e8f6deadded60a135b135520621e25).